core/site.go

```diff
@@ -387,6 +423,143 @@ func (site *Site) sitePower(totalChargePower float64) (float64, error) {
 	return sitePower, nil
 }
 
+// calculates max current per load point
+func (site *Site) applyLoadmanagement(enforce bool) {
+	if !site.lmEnabled {
+		for _, lp := range site.loadpoints {
+			lp.setLMParams(false, 0.0)
+		}
+		return
+	}
+	site.log.TRACE.Printf("--- lm start\n")
+
+	const chargingOffset = 1.1 // when assigning current, apply this offset to let the LP use more current if required
```

Comment:
woher kommt der Wert?

Reply:
Idee: wenn ein LP aus StateB nach StateC wechselt, dann hat er bei aktiviertem LM die Limitation von `minCurrent()`. Um dem LP mehr Strom zu geben, ohne auf Vollast zu gehen, wird pro Interval 10% mehr Strom durch LM erlaubt als `effectiveCurrent()` - bis zum erlaubten Budget oder `maxCurrent()` Nach ca 5-7 Intervallen ist dann der Ausgleich aller LP unter Berücksichtigung ihrer tatsächlich benötigten Ströme erfolgt.
Hintergrund ist, dass ein StateC nicht unbedingt `maxCurrent()` zieht, zB bei Klimatisierung. Der Algo sorgt dafür, dass in diesem Fall die restlichen LP mehr Strom bekommen als (Strom / Nr LP) Gleichverteilung.

Reply:
Warum kann das Lastmanagement nicht gleicht das volle Delta freigeben?

Reply:
Diese Version ist schon nicht mehr enthalten, nach dem Telefonat gabs ein Refactoring.